config/dev.exs

```diff
@@ -1,11 +1,11 @@
 use Mix.Config
 
+database_url =
+  System.get_env("DATABASE_URL") || "ecto://postgres:postgres@localhost/chat_api_dev"
```

Comment:
I made this change so it was easier to test when running with non-local databases (e.g. in my case I run everything on a remote cluster, so i needed to change the hostname while still running in dev mode)

Reply:
nice!